ext/arrow-pycall/extconf.rb

```diff
@@ -12,9 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+require "extpp"
 require "mkmf-gnome2"
 
-unless required_pkg_config_package("arrow-glib")
+unless required_pkg_config_package("arrow-glib",
+                                   debian: "libarrow-glib-dev",
+                                   redhat: "arrow-glib-devel",
+                                   homebrew: "apache-arrow-glib",
+                                   msys2: "arrow")
```

Comment:
We don't need this change because red-arrow does this.

Reply:
OK. I'll revert this..